semgrep-core/tests/OTHER/rules/deep_expr_xml.js

```diff
@@ -0,0 +1,26 @@
+export default function TestComponent4({assets, children, title}) {
+    return (
+      <html lang="en">
+        <head>
+          <meta charSet="utf-8" />
+          <meta name="viewport" content="width=device-width, initial-scale=1" />
+          <link rel="shortcut icon" href="favicon.ico" />
+          <link rel="stylesheet" href={assets['main.css']} />
+          <title>{title}</title>
+        </head>
+        <body>
+          <noscript
+            dangerouslySetInnerHTML={{
+              __html: `<b>Enable JavaScript to run this app.</b>`,
```

Comment:
I think we are missing a `ruleid` here?

Reply:
Never mind me, it seems correct. The test I was thinking of is [tests/js/deep_expr_xml.js](https://github.com/returntocorp/semgrep/pull/5616/files#diff-938523311a9702d086289986a4b74b09cdaaed74898613f271e873345e8aa063).